src/editor-mathfield/pointer-input.ts

```diff
@@ -19,7 +19,12 @@ export function onPointerDown(
 ): void {
   //Reset the atom bounds cache
   mathfield._atomBoundsCache = new Map<string, Rect>();
-
+  if (
+    (evt.target as HTMLElement).tagName === 'MATH-FIELD' &&
+    mathfield.options.readOnly
+  ) {
+    return;
```

Comment:
Is this going to prevent selection of read-only mathfields?

Reply:
the intention was to avoid triggering the re-rendering when you click over a nested mathfield, it didn't work because the main mathfield lose focus which is also causing re-rendering, I need to remove this.